### PR TITLE
Add hotjar code snippet

### PR DESCRIPTION
**Pull request by delapuente**
Fix Qiskit/qiskit.org#296

### CODE REVIEW

File: nuxt.config.js [hunk 1]
```diff
@@ -57,6 +57,18 @@ export default {
...
+        hid: 'hotjar',
+        innerHTML: `
+(function(h,o,t,j,a,r){
+  h.hj=h.hj||function(){(h.hj.q=h.hj.q||[]).push(arguments)};
+  h._hjSettings={hjid:42920,hjsv:6};
+  a=o.getElementsByTagName('head')[0];
+  r=o.createElement('script');r.async=1;
+  r.src=t+h._hjSettings.hjid+j+h._hjSettings.hjsv;
+  a.appendChild(r);
+})(window,document,'https://static.hotjar.com/c/hotjar-','.js?sv=');`
```
Comment (y4izus) [inline nuxt.config.js]: What do you think about putting this script in a separate file and add it as plugin in the `nuxt.config.js` as shown [here](https://github.com/nuxt/nuxt.js/issues/2688) based on [this example](https://nuxtjs.org/faq/google-analytics)

Comment (marinaaisa) [reply nuxt.config.js]: I think @y4izus 's idea is quite good!

**Comment by delapuente** [comment]
@y4izus this is ready for another review.

File: plugins/hotjar.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    hj: any
+    _hjSettings: any
+  }
+}
+
+export default () => {
+  /* eslint-disable */
+  (function(h,o,t,j,a?: any,r?: any){
+    h.hj=h.hj||function(){(h.hj.q=h.hj.q||[]).push(arguments)};
+    h._hjSettings={hjid:42920,hjsv:6};
```
Comment (marinaaisa) [inline plugins/hotjar.ts]: ID for new Hotjar's site for Qiskit.org is different.
You can find it here: https://insights.hotjar.com/sites/1586421/dashboard

So we need to change this ID as suggested below:
```suggestion
    h._hjSettings={hjid:1586421,hjsv:6};
```

**Pull request merged**